projects/collection-editor-library/src/lib/components/asset-browser/asset-browser.component.html

```diff
@@ -21,6 +21,7 @@
           </div>
           <p class="fs-0-986 ml-auto sb-color-grey" >{{assetsCount}}</p>
         </div>
+        <div *ngIf="!myAssets?.length && searchMyInput" class="fs-0785 my-10"> No images found, please try searching for something else </div>
```

Comment:
@santoshilimi don't hard code messages. please add in `label.config.json`

Reply:
Done